test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -360,7 +377,7 @@ public void checkpointWriterInterleavedTest() throws Exception {
         // for the checkpoint. Since regular writes are being interleaved with
         // the checkpointer perfectly, the checkpointer will write on every other
         // address (i.e. odd offsets from the base, startAddress)
-        long startAddress = globalTail.getSequence() + 1;
+        long startAddress = globalTail.getSequence() + 2;
```

Comment:
Please extract `2` as a local variable and provide the explanation why we have +2, what those 2 addresses used for

Reply:
Good idea. Done.